cli/serviceCommands.js

```diff
@@ -0,0 +1,179 @@
+        if (serviceName === '') return;
+
+        let logLevel = args[1];
+
+        if (!logLevel) {
+            logLevel = (
+                await prompts({
+                    type: 'select',
+                    name: 'value',
+                    message: 'Log level',
+                    choices: [
+                        { title: 'TRACE', value: 'trace' },
+                        { title: 'DEBUG', value: 'debug' },
+                        { title: 'INFO', value: 'info' },
+                        { title: 'WARN', value: 'warn' },
+                        { title: 'ERROR', value: 'error' },
+                        { title: 'FATAL', value: 'fatal' }
+                    ],
+                })
+            ).value;
+            if (!logLevel) {
+                console.log('Log level is required');
+                return;
+            }
+        }
+
+        const url = `${config.getConfig('url')}:2002/` + serviceName + '/actuator/loggers/ROOT';
+        const token = config.getConfig('access_token');
+        const body = {configuredLevel: logLevel};
+        const options = {
+            method: 'POST',
+            body: JSON.stringify(body),
+            headers: {
+                'Content-Type': 'application/json',
+                'Authorization': `Bearer ${token}`
+            }
+        };
+
+        let response;
+        try {
+            response = await fetch(url, options);
+        } catch (error) {
+            console.error('Failed to set log level');
+            console.error('Error:', error);
+            return;
+        }
+
+        if (response.ok) {
+            console.error('Log level set successfully');
+        }
+        else
+        {
+            console.error('Failed to set log level');
+            console.error('Response:', response);
+        }
+    },
+
+    async fetchServiceName(args) {
+        let serviceName = args[0];
+
+        if (!serviceName) {
+            serviceName = (
+                await prompts({
+                    type: 'select',
+                    name: 'value',
+                    message: 'Service name',
+                    choices: [
+                        { title: 'Users', value: 'users' },
+                        { title: 'Businessconfig', value: 'businessconfig' },
+                        { title: 'Cards-consultation', value: 'cards-consultation' },
+                        { title: 'Cards-publication', value: 'cards-publication' },
+                        { title: 'External-devices', value: 'external-devices' }
+                    ],
+                })
+            ).value;
+            if (!serviceName) {
+                console.log('Service name is required');
+                return;
+            }
+        }
+        return serviceName;
+    },
+
+    async printHelp() {
+        console.log(`Usage:
+opfab service get-log-level <serviceName>
+or
+opfab service set-log-level <serviceName> <level>
+        `);
```

Comment:
- To be consistent with other services , the content shall be similar to printHelp() of cli/cardCommands.js
- Add the list of possible serviceName  and possible log level 

This gives something like : 

```
Comand list : 
        get-log-level ....

serviceName : card-consultation , card ... 

level : TACE, DEBUG ....


```